src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -26,6 +26,35 @@
 
 namespace Microsoft.ML.Transforms
 {
+    /// <summary>
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="NormalizingTransformer"/>.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | <xref:System.Single> or<xref:System.Double> |
+    /// | Output column data type | The same data type as the input column |
+    ///
+    /// The resulting NormalizingEstimator will normalize the data in one of the following ways based upon how it was created:
+    /// * Min Max - Linear rescale such that the values are mapped to the $[-1,1]$ with the minimum and maximum values being mapped to -1 and 1 respectively.
```

Comment:
>-1,1] [](start = 76, length = 5)

If we not talking about fixZero option all normalizers try to map to [0,1] range.
FixZero is special beast....
Not sure it's worth to specify range...

Reply:
Thanks Ivan - so if its 0,1 unless fixZero is set, why not then just document the normal case? 

---
In reply to: [277070876](https://github.com/dotnet/machinelearning/pull/3432#discussion_r277070876) [](ancestors = 277070876)

Reply:
I have added further description to address this.


---
In reply to: [277084973](https://github.com/dotnet/machinelearning/pull/3432#discussion_r277084973) [](ancestors = 277084973,277070876)